kv-storage/src/rocks_db.rs

```diff
@@ -293,11 +323,35 @@ mod tests {
 
         put_test(&mut db, "key5", "val5").await;
 
+        revert_test(&mut db).await;
+
+        assert!(get_test(&db, "key1", "val1").await);
+        assert!(get_test(&db, "key2", "val2").await);
+        assert!(get_test(&db, "key3", "val3").await);
+        assert!(get_test(&db, "key4", "val4").await);
+        assert!(!get_test(&db, "key5", "val5").await);
```

Comment:
ditto considering https://github.com/postech-dao/simperby/pull/66#issuecomment-1237658987

Reply:
revert_test needs to these lines.